docroot/themes/contrib/civictheme/civictheme_library/components/02-molecules/navigation-card/navigation-card.stories.js

```diff
@@ -59,7 +59,7 @@ export const NavigationCard = (knobTab) => {
       },
       generalKnobTab,
     ), true),
-    modifier_class: text('Additional class', '', generalKnobTab),
+    modifier_class: `story-wrapper-size--medium ${text('Additional class', '', generalKnobTab)}`,
```

Comment:
here and below - this is a leftover fix from the other work. these wrappers are used when the card is shown on the page by itself. But when it is rendered in the Card Container - it should not be wrapped in any additional html. This approach of injecting classes instead of wrapping works well: injected classes rules apply only if classes are preceded by `#root`, which is what happens only one a standalone card page.